safenode/src/network/mod.rs

```diff
@@ -203,12 +203,18 @@ impl SwarmDriver {
             query_interval: Duration::from_secs(5),
             ..Default::default()
         };
-
         let mdns = mdns::tokio::Behaviour::new(mdns_config, peer_id)?;
+
+        let identify_cfg =
+            libp2p::identify::Config::new("safe/0.1.0".to_string(), keypair.public())
```

Comment:
can we pull this from a const? (is this semvar? I wonder how this might interact with the protocol folder and changes there... something for the huddle)

Reply:
Good one, I think for now we should mostly ignore versions. As long as we don't go to major 1. But putting it in a const is probably a good idea.